src/app/shell/admin-tools/data/admins/admins.component.ts

```diff
@@ -118,11 +118,37 @@ export class AdminsComponent implements OnInit, OnDestroy {
     dialogRef.afterClosed().subscribe((result: boolean) => {
       result &&
         this.store.dispatch(
-          new BlockMinistryAdminById(admin.id)
+          new BlockMinistryAdminById({
+            ministryAdminId: admin.id,
+            isBlocked: true
+          })
         );
     });
   }
 
+   /**
+   * This method unBlock Admin By Id
+   */
+    unBlock(admin: UsersTable): void {
+      const dialogRef = this.matDialog.open(ConfirmationModalWindowComponent, {
+        width: Constants.MODAL_SMALL,
+        data: {
+          type: ModalConfirmationType.unBlockMinistryAdmin,
```

Comment:
block/unblock methods seems very similar to me, consider creating one shared method and pass the appropriate message text/action